.eslintrc.js

```diff
@@ -2,27 +2,21 @@ module.exports = {
   env: {
     browser: true,
     es6: true,
-    node: true,
-    mocha: true
+    node: true
   },
   extends: [
     "eslint:recommended",
     "standard"
   ],
-  globals: {
-    Atomics: "readonly",
-    SharedArrayBuffer: "readonly"
```

Comment:
I think I just pulled these from elsewhere, they aren't needed?

Reply:
No, the globals section defines variables that are used from the global scope. (In demo, that would be `hljs` for example.)
Since these two are not used in the code base, there is no need to specify them. (They might get defined in the environments like `window` and `document` would when using `browser` environment.)

See also: https://eslint.org/docs/user-guide/configuring#specifying-globals